test/integration/common/StudioTestHelpers.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+// We need to access the private function Studio.MainWindowColumn, this allows us to.
+// Do not use this outside of tests anywhere. It is extremely dangerous to do so.
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+
+package com.vaticle.typedb.studio.test.integration.common
+
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import com.vaticle.typedb.common.test.core.TypeDBCoreRunner
+import com.vaticle.typedb.studio.Studio
+import com.vaticle.typedb.studio.framework.common.WindowContext
+import kotlinx.coroutines.runBlocking
+
+object StudioTestHelpers {
+    fun studioTest(compose: ComposeContentTestRule, funcBody: suspend () -> Unit) {
```

Comment:
While we're refactoring the tests, I wonder if we can do this in a more intuitive way.

I smell a rat in that the name `studioTest` carries no information. Say we're in a function annotated with `@Test`, located in an integration test suite, for Studio. Why would that, which is clearly a "Studio test" itself, call a function named "studioTest"?

Now, picking apart the implementation. I think adding these nested lambdas that only perform a couple of operations adds quite a bit of indirection which will increase the difficulty of a reader to understand the code being written. Sure, they can copy existing tests and use them as a template, but the ideal state is that they also understand why the code is written as it is.

So what do we actually do here? As I understand it: in the context of a `ComposeRule`, initialise a test window of size 1000x1000, then do whatever work we pass in as a parameter.

Would it, perhaps, be clearer to the reader of the _test itself_, if we broke this up into two parts? 1- create the test window context, 2- run the test.

How would this look like? Perhaps each test would start off with a call to `createTestWindowContext`, and then actually run the test. `funcBody`, after all, does not appear to take in any scope argument. (If I am not mistaken, this method is effectively equivalent to:)
```kt
        runComposeRule(compose) {
            setContent { Studio.MainWindowContent(WindowContext.Test(1000, 1000, 0, 0)) }
        }
        runBlocking { funcBody() }
```

Reply:
Very Kotlinesque, I like it